app/views/miq_request/_request_dialog_details.html.haml

```diff
@@ -28,12 +28,18 @@
       = h(field.values.detect { |k, _v| k == wf.value(field.name) }.try(:last) || wf.value(field.name))
 
     - when "DialogFieldDropDownList"
-      = h(field.value.blank? ? _("<None>") : field.value.to_s.gsub("\x1F", ", "))
+      - if (field.value.class == Array)
+        - values = ""
+        - field.value.map do |a|
+          - values = "#{values}, #{a}"
```

Comment:
How is this variable used?  Seems it's not?

Reply:
I think I forgot to delete that, should be fixed now